catalog/app/app.tsx

```diff
@@ -56,7 +56,7 @@ fontLoader('Roboto', 'Roboto Mono').then(() => {
 const ErrorBoundary = composeComponent(
   'ErrorBoundary',
   Sentry.inject(),
-  createBoundary(({ sentry }) => (error, info) => {
+  createBoundary(({ sentry }: { sentry: any }) => (error: any, info: any) => {
```

Comment:
I should think about it more, but it looks weird right now.
You set "noImplicitAny" to `tsconfig`, then converted `js` to `ts` and set `any` for the value which has exact type.

You need this file as `typescript` because other files depend on it?

Reply:
I suppose, types are:

```
import { _callOnClient } from '@sentry/minimal'
import { CaptureContext } from '@sentry/types'

sentry: typeof _callOnClient
error: Error
info: CaptureContext
```

Reply:
> You set "noImplicitAny" to tsconfig, then converted js to ts and set any for the value which has exact type.

well, it's better be explicit, so we get errors when there's no types, and for now i'm setting untyped things to `any` for compilation to succeed (ima use $TSFixMe for easier tracking)

> Do you need this file as typescript because other files depend on it?

nope, but it's an entry point and it seems logical to start with it

> I suppose, types are:

yep, looks good, but these should be added to the tracking module